Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = ["egui_plot", "demo", "examples/*"]
 [workspace.package]
 edition = "2021"
 license = "MIT OR Apache-2.0"
-rust-version = "1.76"
+rust-version = "1.79"
 version = "0.29.0"
```

Comment:
I think the `egui_plot` version should also be bumped to `0.29.1` to keep it in lockstep with `egui`.

Reply:
nope this is not currently wanted 
The version of egui_plot will be bumped when @emilk creates a new release and not now

Reply:
please see https://github.com/emilk/egui_plot/blob/main/RELEASES.md#versioning